.github/workflows/validation.yml

```diff
@@ -9,6 +9,35 @@ jobs:
   validate-hedera-outputs:
     runs-on: ubuntu-latest
     steps:
+      - name: Setup Java
```

Comment:
This is wrong. All steps done here must be part of the action we provide. This file is just to check / test the action and should only call our action. You can see this file as "This is how an external repo would use it". We do not want external repos to install java / kind /... by hand. With that we need to figure out how we can get those steps now in the new / refactored action